srcpkgs/linux-firmware/template

```diff
@@ -94,3 +95,11 @@ linux-firmware-network_package() {
 		vmove usr/lib/firmware/qca
 	}
 }
+
+linux-firmware-qcom_package() {
```

Comment:
Nit: why `qcom` and not `qualcomm`?

Reply:
Matches the `linux-firmware` directory name and is already done the same way on e.g. [Alpine Linux](https://pkgs.alpinelinux.org/packages?name=linux-firmware-qcom).

Reply:
I can certainly split the FW subpackage to another PR and rebase this, just a few minutes.